requirements-dev.txt

```diff
@@ -1,8 +1,11 @@
+cython
 flake8
 flake8-print
 notebook
 pytest
 pytest-cov
+pytest-cython
 restructuredtext-lint
 sphinx
 sphinx_rtd_theme
+tqdm
```

Comment:
Code in `brainiak` must not use the console. That is why we use logging instead of `print`. You should create a way of passing progress information to the calling code if logging is not enough for you.